packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -23,22 +22,34 @@ export default class CodeMode extends Component<Signature> {
     return realmInfo;
   });
 
-  get backgroundURL() {
+  get realmBackgroundURL() {
     return this.fetchRealmInfo.value?.backgroundURL;
   }
 
-  get backgroundURLStyle() {
-    return htmlSafe(`background-image: url(${this.backgroundURL});`);
+  get realmIconURL() {
+    return this.fetchRealmInfo.value?.iconURL;
+  }
+
+  get realmBackgroundURLStyle() {
+    return htmlSafe(`background-image: url(${this.realmBackgroundURL});`);
   }
 
   <template>
-    <div class='code-mode-background' style={{this.backgroundURLStyle}}></div>
+    <div
+      class='code-mode-background'
+      style={{this.realmBackgroundURLStyle}}
+    ></div>
 
     <div class='code-mode' data-test-code-mode>
       <div class='columns'>
-        <div class='column'>File tree</div>
-        <div class='column'>Code</div>
-        <div class='column'>Schema editor</div>
+        <div class='column column--with-border'>File tree</div>
+        <div class='column column--with-border'>Code</div>
+        <div class='column' data-test-column-card-preview>
+          <CardPreviewPanel
+            @card={{@card}}
```

Comment:
We no longer have a card as an argument for the code mode component so we need a way to fetch it from the `openFile` argument.

Reply:
This is the logic that we have from the old code/ route for doing that. 

https://github.com/cardstack/boxel/blob/416b6ea187ce9e9f6367f4501c6c16220c0667c7/packages/host/app/components/editor/go.gts#L251-L275